assets/css/app.scss

```diff
@@ -0,0 +1,7 @@
+/* This file is for your main application css. */
```

Comment:
Use `//` comments everywhere

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#comment)